tests/test_browser.py

```diff
@@ -2487,18 +2487,20 @@ def test_zzz_emrun(self):
       args_base += ['--browser', browser_path]
       if len(browser_cmd) > 1:
         browser_args = browser_cmd[1:]
-        if 'firefox' in browser_path and '-profile' in browser_args:
+        if 'firefox' in browser_path and ('-profile' in browser_args or '--profile' in browser_args):
           # emrun uses its own -profile, strip it out
           parser = argparse.ArgumentParser(add_help=False) # otherwise it throws with -headless
           parser.add_argument('-profile')
+          parser.add_argument('--profile')
           browser_args = parser.parse_known_args(browser_args)[1]
         if browser_args:
           args_base += ['--browser_args', ' ' + ' '.join(browser_args)]
     for args in [
         args_base,
-        args_base + ['--no_private_browsing', '--port', '6941']
+        args_base + ['--private_browsing', '--port', '6941']
```

Comment:
why did this flip?

Reply:
It was changed to default to non-private browsing because private browsing caused issues on Safari and IndexedDB. Instead of making it a default-on option, to avoid negatives in options, changed it to this form.